src/components/ebay-item-tile-group/item-tile-group.stories.ts

```diff
@@ -0,0 +1,65 @@
+import { buildExtensionTemplate } from "../../common/storybook/utils";
+import Component from "./index.marko";
+import Readme from "./README.md";
+import DefaultTemplate from "./examples/default.marko";
+import DefaultTemplateCode from "./examples/default.marko?raw";
+
+export default {
+    title: "layout/ebay-item-tile-group",
+    component: Component,
+    parameters: {
+        docs: {
+            description: {
+                component: Readme,
+            },
+        },
+    },
+
+    argTypes: {
+        renderBody: {
+            control: { type: "text" },
+        },
+        item: {
+            name: "@item",
+            table: {
+                category: "@attribute tags",
+            },
+            description:
+                "Item to be rendered inside the group. Arguments used are the same as for item-tile. See item-tile for more information.",
+        },
+        columns: {
```

Comment:
Per my previous comment, we can remove this. We just need to impose the layout rules directly in the markup on our own side.